streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -153,6 +153,16 @@
      */
     public static final String CONSUMER_PREFIX = "consumer.";
 
+    /**
+     * Prefix used to differentiate restore consumer configs from {@link KafkaConsumer consumer} configs.
```

Comment:
Nit: we should phrase this this more generic -- compare JavaDoc of `CONSUMER_PREFIX`